spec/features/draws/draw_status_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Draw status' do
+  let(:draw) { FactoryGirl.create(:draw) }
+
+  it 'displays intent metrics' do
+    metrics = { 'off_campus' => 0, 'on_campus' => 2, 'undeclared' => 1 }
+    create_intent_data(draw, metrics)
+
+    log_in(FactoryGirl.create(:user))
+    visit draw_path(draw)
```

Comment:
as a note, we should standardize using path helpers vs using url strings in integration tests; I think that the path helpers are better so we should use them moving forward